### ISSUE TITLE
Keyboard navigation with arrow / cursor keys between icons does not work

### ISSUE THREAD

**Issue by sdbbs** [DEM0NAssissan7/desktop-icons-neo#48]
Just tried this extension, and like the NG, keyboard navigation between icons does not work. Example: click to select a single desktop icon; press Left Arrow key on keyboard to select the desktop icon to the left of currently selected one; result: nothing happens, selection remains on the desktop icon previously selected with the mouse.

My settings

```
$ cat /etc/issue
Ubuntu 22.04.4 LTS \n \l

$ gnome-shell --version
GNOME Shell 42.9

$ gnome-extensions info desktopicons-neo@darkdemon
desktopicons-neo@darkdemon
  Name: Desktop Icons: Neo
  Description: This adds desktop icons to GNOME. A fork of Desktop Icons NG with a massive amount of customizations (like icon shapes and curved corners), features, bug fixes, and overall polish.
  Path: /home/user/.local/share/gnome-shell/extensions/desktopicons-neo@darkdemon
  URL: https://github.com/DEM0NAssissan7/desktop-icons-neo
  Version: 8
  State: ENABLED
```
